service/history/historyEngine.go

```diff
@@ -2713,9 +2750,80 @@ func (e *historyEngineImpl) ReapplyEvents(
 				reapplyEvents,
 			); err != nil {
 				e.logger.Error("failed to re-apply stale events", tag.Error(err))
-				return nil, &workflow.InternalServiceError{Message: "unable to re-apply stale events"}
+				return emptyUpdateAction, &workflow.InternalServiceError{Message: "unable to re-apply stale events"}
 			}
 
 			return postActions, nil
 		})
 }
+
+func (e *historyEngineImpl) loadWorkflowHelper(
```

Comment:
sorry but can you plz rename this function, e.g. helper does not mean anything

Reply:
Renamed to loadWorkflowOnce